tests/test_electric_source_sink.py

```diff
@@ -46,6 +48,9 @@ def test_source_sink(self):
         results = solution.extract_results()
         parameters = solution.parameters(0)
 
+        # Test energy conservation
+        electric_power_conservation_test(solution, results)
```

Comment:
update docstring: if there are additional items being test in the utils function

Reply:
This check was already there, but then done explicitly. I removed the manual check and replaced it by this automated one.